ReleaseNotes.md

```diff
@@ -4,7 +4,7 @@
 
 ### Changes in Viper Language
 
-- Quasihavoc: New statements ``quasihavoc`` and ``quasihavocall`` allow explicitly losing information about a resource. E.g., ``quasihavoc c ==> P(x)`` havocs the snapshot of resource ``P(x)`` if ``c`` is true and some permission to ``P(x)`` is held, and does nothing otherwise. Semantically equivalent to (conditionally) exhaling the current permission amount to the given resource and inhaling it again, but implemented more efficiently. For more details, see [this report](https://ethz.ch/content/dam/ethz/special-interest/infk/chair-program-method/pm/documents/Education/Theses/Daniel_Zhang_PW.pdf) ([Silver#611](https://github.com/viperproject/silver/pull/611)). Currently, Silicon fully supports ``quasihavoc`` and ``quasihavocall``, while Carbon only supports ``quasihavoc`` for field and predicate resources.
```

Comment:
I'm also not sure whether the last sentence means Carbon doesn't support quasihavocall at all?

Reply:
Yes, Carbon does not support `quasihavocall` at all. We could rewrite the sentence to:

"Currently, Silicon fully supports ``quasihavoc`` and ``quasihavocall``, while Carbon only supports ``quasihavoc`` for field and predicate resources and does not suport ``quasihavocall``."

Reply:
sounds good to me!

Reply:
Sounds good, I'll add that change tomorrow.